tests/integration/bond_test.py

```diff
@@ -55,6 +56,38 @@ def setup_remove_bond99():
     netapplier.apply(remove_bond)
 
 
+@contextmanager
+def bond_interface(name, slaves=[]):
```

Comment:
1. I was under the impression that you do not want to have defaults for slaves.
2. It is not allowed to have mutable defaults, you can only use `None` or a tuple for that.

I am in favour of having no default.

Reply:
Test without cleanup now using context manager as well.
Test which using 'setup_remove_bond99' will be fixed in other patch ( bond dict includes ip and options)

Reply:
Default was added by mistake .